lib/routes/authentication/onBoarding.dart

```diff
@@ -44,6 +45,17 @@ Widget renderPending() {
   );
 }
 
+void showErrorMessage(String message) {
+  Fluttertoast.showToast(
+      msg: message,
+      toastLength: Toast.LENGTH_SHORT,
+      gravity: ToastGravity.SNACKBAR,
+      timeInSecForIosWeb: 1,
+      backgroundColor: Colors.black54,
```

Comment:
Let's store this color in tiler styles file so we can update schemes in the future,